jcommune-service/src/test/java/org/jtalks/jcommune/service/transactional/TransactionalUserServiceTest.java

```diff
@@ -503,11 +505,25 @@ public void testLoginUserWithBadCredentialsShouldFail()
         HttpServletResponse httpResponse = new MockHttpServletResponse();
         LoginUserDto loginUserDto = new LoginUserDto("", "password", true, "192.168.1.1");
         when(authenticator.authenticate(loginUserDto, httpRequest, httpResponse))
-                .thenReturn(false);
+                .thenReturn(AuthenticationStatus.AUTHENTICATION_FAIL);
 
-        boolean result = userService.loginUser(loginUserDto, httpRequest, httpResponse);
+        AuthenticationStatus result = userService.loginUser(loginUserDto, httpRequest, httpResponse);
 
-        assertFalse(result, "Login user with bad credentials should fail.");
+        assertEquals(result, AuthenticationStatus.AUTHENTICATION_FAIL, "Login user with bad credentials should fail.");
+    }
+
+    @Test
+    public void testLoginNotActivatedUserShouldFail()
+            throws UnexpectedErrorException, NoConnectionException {
```

Comment:
It doesn't matter in tests what exceptions get thrown, so it's easier to `throws Exception`.